tests/snappi/pfc/files/helper.py

```diff
@@ -1,19 +1,24 @@
 import time
 import logging
+import uuid
 from tests.common.helpers.assertions import pytest_assert
 from tests.common.fixtures.conn_graph_facts import conn_graph_facts,\
     fanout_graph_facts
 from tests.common.snappi.snappi_helpers import get_dut_port_id
 from tests.common.snappi.common_helpers import pfc_class_enable_vector,\
-    get_egress_lossless_buffer_size, stop_pfcwd, disable_packet_aging
+    get_egress_lossless_buffer_size, get_ingress_lossless_buffer_size,\
+    get_port_prio_dropped_packets, stop_pfcwd, disable_packet_aging
 from tests.common.snappi.port import select_ports, select_tx_port
 from tests.common.snappi.snappi_helpers import wait_for_arp
 
 logger = logging.getLogger(__name__)
 
+unique_end = None
+flow_port_config = []
 PAUSE_FLOW_NAME = 'Pause Storm'
 TEST_FLOW_NAME = 'Test Flow'
 TEST_FLOW_AGGR_RATE_PERCENT = 45
+TEST_FLOW_SINGLE_RATE_PERCENT = 95
```

Comment:
This is the value used in tests to generate traffic (traffic flow rate from Tx port) and has been used to calibrate the pfc response delay values